internal/pullbox/tar.go

```diff
@@ -59,7 +60,9 @@ func (p *pullbox) copy(overwrite bool, src, dst string) error {
 
 	if !overwrite {
 		for _, srcFile := range srcFiles {
-			if _, err := os.Stat(filepath.Join(dst, srcFile.Name())); err == nil {
+			// Only show error if file exists and is not a non-modified config
```

Comment:
should this be:
- is a non-modified config
- OR: is not a modified config
?

Reply:
ugh, this is wrong (on several levels). I'll fix.